client/my-sites/upgrades/checkout/checkout.jsx

```diff
@@ -306,12 +310,18 @@ const Checkout = React.createClass( {
 	}
 } );
 
+function mapStateToProps ( state ) {
+	const cards = getStoredCards( state );
+	const selectedSite = getSelectedSite( state );
```

Comment:
I was wondering about this! I should have asked earlier but saw that the sites.getSelectedSite was always assumed to give results :P
I'm always in favour for of bulletproofing so I'll make some changes soon :D

Ohh and good point on the more targeted selectors! The more targeted the better! 👌